### PR TITLE
Uniform settings capitalization

### PR DESCRIPTION
**Pull request by ajmeese7**
All the settings with the exception of `Font color Style` shared a sentence style of capitalization, so that label has been changed to match the others.

### CODE REVIEW

**Comment by andersevenrud** [comment]
I might be blind, but as far as I can see the other instances of "style" is the first word of the sentence, which always should be capitalized.

**Comment by ajmeese7** [comment]
That's my point, the first letter is the only one capitalized in all of the settings with the exception of this one. Hence why I changed the sentence capitalization to match the formatting of all the other setting labels.

**Comment by andersevenrud** [comment]
I'm a bit confused here. According to the patch you changed the *last* word of the sentence.

**Comment by ajmeese7** [comment]
Let me rephrase: all of the settings labels follow a sentence case, i.e. the first word is capitalized and none of the other words are. For this setting, that rule was not followed because the last word of the sentence was also capitalized, so the appearance does not match the casing of the rest of the labels.

Examples of other settings:
- "Enable desktop icons"
- "Custom font color"

The setting I changed:
- "Font color Style" => "Font color style"

For a more uniform appearance of the settings labels.

**Comment by andersevenrud** [comment]
> I might be blind

Apparently I am :man_facepalming: I somehow read this the other way around...

**Review by andersevenrud — APPROVED** [review]

**Comment by ajmeese7** [comment]
All good friend, it happens to the best of us. Thanks for approving! I just want the OS.js organization as a whole to be as polished as possible, so I'm trying to catch the little things here and there to do my part.

**Comment by andersevenrud** [comment]
This was published under a 1.2.0 release tag.
